src/ResourceManager/Websites/Commands.Websites/Cmdlets/WebApps/NewAzureRmWebAppAzureStoragePath.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.Azure.Commands.WebApps.Models;
+using Microsoft.Azure.Management.WebSites.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Commands.WebApps.Cmdlets.WebApps
+{
+    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "WebAppAzureStoragePath", SupportsShouldProcess = true), OutputType(typeof(WebAppAzureStoragePath))]
```

Comment:
@vinisoto as a note, it's not necessary to support `ShouldProcess` in this scenario since you are creating an in-memory object, but it doesn't hurt to have it supported 😀